src/logger/loggerSampleData.c

```diff
@@ -175,34 +175,6 @@ float get_analog_sample(int channelId)
 }
 #endif
 
-#if TIMER_CHANNELS > 0
```

Comment:
It was moved into timer_config.c.  In there the pre-processor directive isn't needed since the code will be discarded by the linker if not used.